tests/behat/features/contrib/config_readonly.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Contrib - config_readonly
+  In order to verify that contrib module config_readonly is working
+  As an administrative user
+  I should be not be able to administer settings when the module is active
+
+  @api
+  Scenario: Ensure no configuration changes are possible through the UI.
+    Given I am logged in as a user with the "Administrator" role
+    Then check that the "config_readonly" module is enabled
```

Comment:
wouldn't you wanna check that a form is locked. just because the module is enabled, doesn't actually lock any configs.

Reply:
I started down that path but due to the part that we don't lock config in CI environments that made it a blocker. 

The flag for enabling the locking is in the environment settings `$settings` variable which is read only and set at bootstrap. This makes it nearly impossible to set to TRUE after bootstrap. 

I am open to suggestions.

Reply:
hmm. good point. think we'll come back to that later when we get profile CI/CD set up. then we can set the setting in something like https://github.com/SU-SWS/acsf-cardinalsites/blob/8.x-1.x/docroot/sites/settings/ci.settings.php